kie-remote/src/main/java/org/kie/remote/util/LocalMessageSystem.java

```diff
@@ -33,7 +37,9 @@ private LocalMessageSystem() { }
     }
 
     public void put(String topic, Object message) {
-        queueForTopic(topic).offer( message );
+        if (!queueForTopic(topic).offer(message)) {
+            logger.info("msg :{} not added in the topic:{}", message, topic);
```

Comment:
I would say `error` more than `info` :)
Isn't better to throw an exception in this case?

Reply:
Is used for localtest, ok for runtime exception